cpp/dolfinx/io/pyvistawriter.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2020 Garth N. Wells
```

Comment:
I don't think we should have these files. They key is returning the raw data in a flexible enough format that pyvista can process it.

Reply:
This is why I first used numba, but moved everything into C++ to avoid getting numba as a dependency.